### DISCUSSION TITLE
Is there a function like regulon_specificity_scores in pySCENIC?

### DISCUSSION THREAD

**Discussion by Sophia409** [aertslab/SCENIC#268] [Python]
Hello aertslab,
Wondering how can I compute regulon_specificity_scores to get cell type specific regulons in SCENIC? Is there a function like regulon_specificity_scores in pySCENIC?
![image](https://user-images.githubusercontent.com/30619010/71081781-e65a2800-21ca-11ea-8ed0-a4b4c73c8fe9.png)
![image](https://user-images.githubusercontent.com/30619010/71081838-f6720780-21ca-11ea-90bb-79164acf7e4e.png)

**Comment by gloriafight** [comment]
I also want to use this function. The following is my methods. The file mac_all_auc_mtx.txt from pyscenic aucell result.
# regulons activaty by cluster
auc <- read.csv("./v2/mac_all_auc_mtx.txt",row.names = 1)
colnames(auc) <- gsub("\\.{3}","",colnames(auc))
AUCmatrix <- t(auc)
regulonActivity_byCellType <- sapply(split(cell_cluster$cellid, cell_cluster$cluster),
                                     function(cells) rowMeans(AUCmatrix[,cells]))

**Reply by apal6** [reply]
Hi, for some reason I am not able to run this function.

error:
`error in evaluating the argument 'x' in selecting a method for function 'rowMeans': could not find function "getAUC"`

Could you help? Thanks
